packages/n-back/README.md

```diff
@@ -0,0 +1,71 @@
+# n-back
+
+## Overview
+
+A timeline for the nback task
+
+## Loading
+
+### In browser
+
+```html
+<script src="https://unpkg.com/@jspsych-timelines/n-back">
+```
+
+### Via NPM
+
+```
+npm install @jspsych-timelines/n-back
+```
+
+```js
+import { createTimeline } from "@jspsych-timelines/n-back"
+```
+
+## Compatibility
+
+`@jspsych-timelines/n-back` requires jsPsych v7.0.0 or later.
```

Comment:
would it be possible to turn this timeline into supporting v8 immediately?

Reply:
I think I didn't use any functions that were affected by the change from v7 to v8, so I'll probably just update the README